integration_tests/README.md

```diff
@@ -26,6 +93,17 @@ Where possible, targets are being run in docker containers (this works for Postg
 
 ### Creating a new integration test
 
+#### Set up profiles
+Do either one of the following:
+1. Use `DBT_PROFILES_DIR`
+    ```shell
+    cp integration_tests/ci/sample.profiles.yml integration_tests/profiles.yml
+    export DBT_PROFILES_DIR=$(cd integration_tests && pwd)
+    ```
+2. Use `~/.dbt/profiles.yml`
+    - Copy contents from `integration_tests/ci/sample.profiles.yml` into `~/.dbt/profiles.yml`.
```

Comment:
When given the choice, I prefer including a `profiles.yml` file within each dbt project and then setting `DBT_PROFILES_DIR`. See example within `run_test.sh`

Of course this relies on `profiles.yml` to be completely devoid of actual secrets (which seems eminently doable since we can use environment variables instead).

Any cautions for me before I rename `integration_tests/ci/sample.profiles.yml` to `integration_tests/profiles.yml`?